src/lints/inherent_method_must_use_added.ron

```diff
@@ -25,42 +26,43 @@ SemverQuery(
                                 method_visibility: visibility_limit @filter(op: "=", value: ["$public"]) @output
                                 method_name: name @tag @output
 
-                                attribute @fold @transform(op: "count") @filter(op: "=", value: ["$zero"]) {
+                                attribute {
+                                    new_attr: raw_attribute @output
                                     content {
                                         base @filter(op: "=", value: ["$must_use"])
                                     }
                                 }
+
+                                span_: span @optional {
+                                    filename @output
+                                    begin_line @output
+                                }
                             }
                         }
                     }
                 }
             }
-            current {
+            baseline {
                 item {
                     ... on ImplOwner {
                         visibility_limit @filter(op: "=", value: ["$public"])
                         name @filter(op: "=", value: ["%name"])
+                        __typename @filter(op: "=", value: ["%owner_type"])
 
                         importable_path {
                             path @filter(op: "=", value: ["%path"])
-                        }
+                        }                        
```

Comment:
```suggestion
                        }
```
You should be able to configure your editor to strip trailing whitespace like this automatically. It's nice so that other people opening the file don't end up accidentally editing it when _their_ editor strips the whitespace introduced by someone else.

Reply:
I applied the change and also added source control recognizing such changes instead of ignoring them. Sorry for making you correct such mistakes, this should not happen again now.